cli/lib/kontena/cli/stacks/registry/update_command.rb

```diff
@@ -0,0 +1,29 @@
+require_relative '../common'
+
+module Kontena::Cli::Stacks::Registry
+  class UpdateCommand < Kontena::Command
+    include Kontena::Cli::Common
+    include Kontena::Cli::Stacks::Common
+    include Kontena::Cli::Stacks::Common::RegistryNameParam
+
+    banner "Changes stack settings on registry"
+
+    option '--make-private', :flag, "Change visibility to private"
```

Comment:
Nope. `make-something` is clear indicator that this should be included in command name.